diff -Naur opensched/src/sched.cc opensched-bug/src/sched.cc
--- opensched/src/sched.cc	Mon Jun 18 21:42:10 2001
+++ opensched-bug/src/sched.cc	Mon Feb  4 21:48:24 2002
@@ -317,7 +317,7 @@
 
     if ( t->block() )
     {
-	while ( ! r->reverseIsAvailable(tfinish-t->remaining+1, tfinish) )
+	while ( ! r->reverseIsAvailable(tfinish-t->rremaining+1, tfinish) )
 	    --tfinish;
 	ReverseBookResource(t, NULL, r, tfinish-t->rremaining+1, tfinish,
 			    TimeBlock::AUTO_SCHEDULE);
